app/build.gradle

```diff
@@ -72,6 +72,7 @@ dependencies {
       "androidx.fragment:fragment:$fragment_version",
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
       'androidx.multidex:multidex:2.0.1',
+      'androidx.preference:preference:1.1.0-alpha02',
```

Comment:
As per this link: https://developer.android.com/jetpack/androidx/releases/preference

The latest version is `1.1.0`, so please update this.

Reply:
Updated.